### ISSUE TITLE
Please make static the temp path to akroma.exe 

### ISSUE THREAD

**Issue by oldominion** [akroma-project/akroma-wallet-desktop#25]
Is there any chance you could make the temp path of the akroma.exe file static? My application firewall doesn't allow exclusions by process name, so i'm having to allow it access each time,

**Comment by detroitpro** [comment]
Yes. absolutely. The current version is configured as a 'portable' application, in the next few releases we will have a full installer.

Leaving this issue open until we have a full installer.
